packages/reputation-miner/viz/repCycle.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Colony Mining Cycle Visualizer</title>
+  </head>
+  <body>
+
+    <!-- Load Dependencies -->
+    <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
+    <script src="https://cdn.ethers.io/scripts/ethers-v4.min.js"></script>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+
+    <!-- Page HTML -->
+    <div id="app">
+      <h2>Welome to RepMinCycViz!</h2>
+      <h3>Mining Cycle Address:</h3>
+      <input placeholder="Mining Cycle Address" v-model="cycleAddress">
```

Comment:
Will this work across mining cycles? i.e. when the cycle completes and the `cycleAddress` changes?

Reply:
Currently you have to manually update the mining cycle address. Over the weekend I was thinking of ways to have the visualizer automatically fetch the address -- doing the fetch on load should be no problem, but automatically refreshing when the cycle updates would be a bit more involved. At minimum refreshing the visualizer should give you the current mining cycle.

Reply:
In #596 we had a similar requirement and ended up [listening for the `ReputationMiningCycleComplete` event](https://github.com/JoinColony/colonyNetwork/pull/596/files#diff-86b0e8d40084ff0b818614beb26393bfR95). These 2 PRs are very close in that they improve the quality of life of reputation miners so perhaps we can rebase this on top of #596 and hook into that event handler as well?

Reply:
Mm I'll look into it. It may not be necessary though, since unlike the miner which is meant to be running automatically in the background, users will generally be interacting with the visualizer manually, so a manual refresh (especially given that mining cycles are an hour+) doesn't seem too bad. Although I could imagine someone potentially wanting to leave the visualizer running on a screen for a longer period of time.

Reply:
Although I'd still say that we should merge this as-is and we can add the auto-update functionality in a later PR.